test/puppetlabs/puppetdb/examples.clj

```diff
@@ -5,7 +5,7 @@
    {:certname         "empty.catalogs.com"
     :api_version      1
     :version          "1330463884"
-    :transaction_uuid nil
+    :transaction_uuid "aaaaaaaa-1111-aaaa-1111-aaaaaaaaaaaa"
```

Comment:
Does changing this from nil have implications & coverage concerns for other tests now? I mean, right now we're testing a blank tuuid in other places, and this patch means we aren't any more. nil is still valid right?